gapic/api/artman_errorreporting.yaml

```diff
@@ -0,0 +1,28 @@
+common:
+  api_name: google-devtools-clouderrorreporting-v1beta1
+  import_proto_path:
+    - ${THISDIR}/../..
+  src_proto_path:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1
+  service_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/errorreporting.yaml
+  gapic_api_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1/errorreporting_gapic.yaml
+  auto_merge: true
+  auto_resolve: true
+  ignore_base: false
+  output_dir: ${REPOROOT}/artman/output
+java:
+  final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-errorreporting
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-errorreporting
+go:
+  final_repo_dir: ${REPOROOT}/gapi-errorreporting-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-errorreporting
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-errorreporting
+ruby:
+  final_repo_dir: ${REPOROOT}/gcloud-ruby/google-cloud-errorreporting
+nodejs:
+  final_repo_dir: ${REPOROOT}/gcloud-node/packages/gcloud-node-errorreporting
```

Comment:
gcloud-node's naming policy is actually:
- package name: `@google-cloud/<API_NAME>`
- directory: `packages/<API_NAME>`

So here should be `${REPOROOT}/gcloud-node/packages/errorreporting`, `gcloud-node-` part is not necessary.